src/shared/options/readOptions.ts

```diff
@@ -44,6 +45,10 @@ export async function readOptions(
 		tokens: true,
 	});
 
+	if (mode === "migrate" && !values.base) {
+		values.base = await getBase();
+	}
```

Comment:
[Style] Nit: if you really wanted to, this could instead be `base: values.base || (mode === "migrate" && (await getBase())),` a few lines down... No strong feelings either way on my end 😄, just raising in case you like it!

Reply:
Eh, I don't find that quite as readable. I know mutation is often taboo, but I feel like it's reasonable here.

Reply:
👍 a wontfix it is! Makes sense